amulet/deployer.py

```diff
@@ -123,7 +124,7 @@ def load(self, deploy_cfg, deployment_name=None):
                 or next(iter(deploy_cfg.values()))
         self.series = schema.get('series', self.series)
         self.relations = schema.get('relations', [])
-        for service, service_config in schema['services'].items():
+        for service, service_config in schema['applications'].items():
```

Comment:
I think we need to check for schema['services'] or schema['applications'] one line above this. Something like this:
```python
if schema.get('applications'):
    services = schema['applications']
else:
    services = schema['services']
for service, service_config in services.items():
```